docker-compose.yml

```
@@ -70,6 +70,24 @@ services:
       - SC_WEB_URL=http://web:8000
     depends_on:
       - web
+
+  py-sc-server:
+    build:
+      context: ./problem-solver/py
+      dockerfile: ./Dockerfile
```

```suggestion
      context: ./
      dockerfile: ./problem-solver/py/Dockerfile
```
This is required to use `run_py_sc_server.sh` inside the py-sc-server container